xml/Microsoft.Extensions.Primitives/StringSegment.xml

```diff
@@ -556,7 +558,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets whether this <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> contains a valid value.</summary>
-        <value>To be added.</value>
+        <value>True if the <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> contains a valid value.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> contains a valid value; <see langword="false" /> otherwise.</value>
```